.github/workflows/tests.yml

```diff
@@ -96,32 +96,12 @@ jobs:
       - name: Run E2E tests against ${{ matrix.framework }} example
         run: yarn workspace e2e test
         env:
-          # Override on the default value in `cypress.json` with `@react and not @skip`
-          TAGS: '@${{ matrix.framework }} and not @skip'
```

Comment:
Let's move to using `@skip` for local testing only. If we want to merge tests that aren't run for a specific framework, then we should just leave that framework's tag off the test.

Reply:
I like this! For running tests locally in dev, how would I set this environment variable?

`TAGS=@angular yarn e2e dev`?

Reply:
You can do `yarn e2e dev -e "TAGS=@angular and not @skip"`. And to run the exact script CI runs you can do `yarn e2e test -e "TAGS=@angular"`

Reply:
I agree that this is the right decision long-term.

`@focus` is what I use locally, but `@skip` is also a helpful tool in the toolbelt.

The only problem with "leave that framework's tag off the test" is that it means _we won't show content for unsupported use-cases_.

This is where the dilemma is:

1. `.feature`s describe supported use-cases
2. The `@react` framework tag indicates that this is _supported behavior for React_ (React & Next.js aren't always 1-to-1)
3. But, we recognize that some behavior **cannot be tested in CI**.

So it sounds like we may need an alternative solution here to differentiate those two – perhaps with an explicit `@skip-ci` tag?

Reply:
I guess it leads to the question of: do we want to have `.feature` files that will never translate to a working test? And if not, how do we want to include those features in our docs?

Reply:
What's something that will "never translate to a working test"?

My take is we want to codify all behavioral expectations because this is the one source of truth for our behavior that's consistent between product, engineering, and customer expectations.

So if something can't be tested for a technical reason, then at least it's documented, tracked, & can be unblocked one-day.